i18n/en.toml

```diff
@@ -109,3 +109,21 @@ other = "Japan users"
 
 [graph_percent_https]
 other = "Percent of Pageloads over HTTPS (14 day moving average)"
+
+[ct_name]
```

Comment:
Maybe you can add a comments to specify that those words related to certificate transparency? or explicit the name: `[certificate_transparency_name]`

Reply:
Thanks for the suggestion, I changed all keys to start with `certificate_transparency`. Should be clear enough, right?